GATEWAY/src/providers/mqttClient.ts

```diff
@@ -1,7 +1,7 @@
 import { Injectable } from '@angular/core';
 import { BackgroundFetch } from '@ionic-native/background-fetch';
 import { Events } from 'ionic-angular';
-import mqtt from 'mqtt';
+import * as mqtt from 'mqtt';
```

Comment:
Does not work when i try to test it
This version of the import works, and does not influence the code in any way
Just for testing purposes